src/Commands/InstallCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Rapidez\Statamic\Commands;
+
+use Illuminate\Console\Command;
+
+class InstallCommand extends Command
+{
+    protected $signature = 'rapidez-statamic:install';
+
+    protected $description = 'Install Rapidez Statamic';
+
+    public function handle(): void
+    {
+        $this->call('vendor:publish', [
+            '--tag' => 'runway-config',
+        ]);
+
+        $this->call('vendor:publish', [
+            '--provider' => 'Statamic\Eloquent\ServiceProvider',
+            '--tag' => 'statamic-eloquent-config',
+        ]);
+
+        $this->confirm('Did you set the table_prefix in config/statamic/eloquent-driver.php to "statamic_"?');
+
+        $this->call('statamic:install:eloquent-driver');
+
+        $this->info('If you would like to store users in the database please make sure to follow this guide: https://statamic.dev/tips/storing-users-in-a-database#from-a-fresh-statamic-project');
+
+        if ($this->confirm('Would you like to configure a multisite?')) {
```

Comment:
I remember a something a while ago when a site needed to be multisite to work correct, is this still te case? Do we really support both?

Reply:
Multisite would make it that the user has to buy the PRO version of Statamic. Noted and will look into using a non multisite setup later!